### PR TITLE
Duo Mapping Enrichment

### PR DESCRIPTION
**Pull request by eepstain**
## Contributing to Cortex XSOAR Content
Make sure to register your contribution by filling the [contribution registration form](https://forms.gle/XDfxU4E61ZwEESSMA)

**The Pull Request will be reviewed only after the contribution registration form is filled.**

## Status
- [x] In Progress
- [ ] Ready
- [ ] In Hold - (Reason for hold)

## Related Issues
fixes: [link to the issue](https://jira-hq.paloaltonetworks.local/browse/CIAC-7989)

## Description
A few sentences describing the overall goals of the pull request's commits.

## Must have
- [ ] Tests
- [ ] Documentation

### CODE REVIEW

**Comment by ShirleyDenkberg** [comment]
@yasta5 Looks good.  Doc review completed.

File: Packs/DuoAdminApi/ReleaseNotes/4_0_10.md [hunk 1]
```diff
@@ -0,0 +1,12 @@
+
+#### Modeling Rules
+
+##### New: Duo 2.0 Modeling Rule
+
+Created a new Modeling Rule version (Available from Cortex XSIAM 2.0).
+
+##### Duo Modeling Rule
+
+Added a Out-of-the-box analytics for XDM identity data.
```
Comment (ShirleyDenkberg) [inline Packs/DuoAdminApi/ReleaseNotes/4_0_10.md:10]:
```suggestion
Added out-of-the-box analytics for XDM identity data.
```

File: Packs/DuoAdminApi/README.md [hunk 2]
```diff
@@ -0,0 +1,2 @@
+# Duo Admin
+This pack includes Cortex XSIAM content.
```
Comment (ShirleyDenkberg) [inline Packs/DuoAdminApi/README.md:2]: Don't you need some additional information here?

**Comment by ShirleyDenkberg** [comment]
@evisochek Doc review completed.

File: Packs/DuoAdminApi/README.md (hunk 2 above)
Comment (eepstain) [reply Packs/DuoAdminApi/README.md:2]: In this pack's specific case- No, since it's with an Event Collector integration.

**Review by evisochek — APPROVED** [review]

**Pull request merged**